app/controllers/idv/capture_doc_controller.rb

```diff
@@ -14,6 +14,13 @@ class CaptureDocController < ApplicationController
       analytics_id: Analytics::DOC_AUTH,
     }.freeze
 
+    def return_to_sp
+      redirect_to return_to_sp_failure_to_proof_url(
+        step: next_step,
+        **params.permit(:location).to_h.symbolize_keys,
```

Comment:
if we're only permitting one key... should we just do that one key?

```suggestion
        location: params[:location],
```

Reply:
I forget if we need to permit individual keys (IIRC it was just for iterating that we need to) but otherwise we could do this:

```suggestion
        location: params.permit(:location)[:location],
```

Reply:
Updated in 82d7f70.